mappers/lodging.js

```diff
@@ -61,7 +61,7 @@ export default function mapLodgings(records, translations) {
       if (type) {
         publicG.add(sym(lodgingUri), SCHEMA('keywords'), sym(type));
       } else {
-        console.error(`Cannot map subtype value '${record['sub_type']}' for record ${recordId}`);
+        errorLogger('subtype', record['sub_type'], recordId);
```

Comment:
Pass the same field name (`sub_type`) as used to lookup the value as first argument. I know it was different before, but old code is wrong.